scripts/publish-local.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-TARGET=$"/Users/talha.naqvi/Documents/Work/RLV-Demo/node_modules/recyclerlistview/dist" #target-path
+TARGET=$"/Users/andrii.tiertyshnyi/Documents/Projects/bolt-delivery-provider-app/node_modules/recyclerlistview/dist" #target-path
```

Comment:
in future, we can use some env params here

Reply:
🤦‍♂